internal/ethapi/api.go

```diff
@@ -998,6 +998,7 @@ type BlockOverrides struct {
 	Coinbase   *common.Address
 	Random     *common.Hash
 	BaseFee    *hexutil.Big
+	BlobFee    *hexutil.Big
```

Comment:
```suggestion
	BlobBaseFee    *hexutil.Big
```
Sorry to be pedantic here, but I think it would be good to update throughout. "`BlobFee`" could be construed as the fee for including a single blob, whereas this value is actually the price per blob gas of blobs in this block.